indexing-service/src/test/java/org/apache/druid/indexing/common/task/CompactionTaskTest.java

```diff
@@ -428,7 +432,11 @@ public void testCreateCompactionTaskWithConflictingGranularitySpecAndSegmentGran
   @Test
   public void testCreateCompactionTaskWithTransformSpec()
   {
-    ClientCompactionTaskTransformSpec transformSpec = new ClientCompactionTaskTransformSpec(new SelectorDimFilter("dim1", "foo", null));
+    ClientCompactionTaskTransformSpec transformSpec = new ClientCompactionTaskTransformSpec(new SelectorDimFilter(
```

Comment:
Super nit: It would be easier to read if the whole of `new SelectorDimFilter(...)` was in a single new line.

Reply:
Agreed. I changed the line break since I had to make changes to the imports anyway for checkstyle.